### DISCUSSION TITLE
Real Basic Question on Git Bash

### DISCUSSION THREAD

**Discussion by chris12dec** [community/community#22727] [New to GitHub]
<p>Using Git Bash, inside my Windows My Documents folder, I just typed git init, when I should have first changed directory to the new git_practice directory I had created.  I’m worried about what effect this is going to have on the My Documents folder and its contents, e.g. somehow overwrite everything, although that hasn’t happened yet!</p>
<p>Can someone tell me please how to “un-initialise” My Documents?  And also, will I experience any problems following my mistake?</p>
<p>I have been following a Codecademy course and this is how I got into this trouble.</p>
<p>Thanks for any help you can give.</p>

**Comment by chris12dec** [comment]
<p>I have done some research and it appears all “git init” does is create the hidden file .git</p>
<p>So I have to delete it using “rm -rf .git”</p>
<p>Does that sound correct?  Thanks for any help.</p>

**Comment by gwenKneeland** [comment, marked as answer]
<p>That should take care of it yes. Keep in mind that <code>git init</code> only creates that hidden <code>.git</code> folder. Unless you used <code>git add ...</code> <strong>and</strong> <code>git commit ...</code> nothing would have been added to that repository at all. Cheers!</p>
